scripts/install.js

```diff
@@ -0,0 +1,26 @@
+const { execSync } = require('child_process');
+
+const installCmd = process.env.npm_config_refer === 'ci' ? 'npm ci' : 'npm install';
```

Comment:
Yeah, pretty much. Currently the flow is like

dashboard-e2e -> dashboard -> api-server

They all depend on the "root" package as well, which installs the git hooks and prettier etc. The requirement for ros2/rmf is kind of implied by the requirement on `rmf_core` and `traffic_editor`. When we move the ros2 plugin out of tree, it would be like

dashboard-e2e -> dashboard
dashboard-e2e -> api-server-plugin-ros2
dashboard -> api-server
api-server
api-server-plugin-ros2 -> api-server

So installing dashboard no longer needs ros2/rmf.